interfaces/matlab/Utility/+ct/+impl/call.m

```diff
@@ -1,13 +1,13 @@
-function output = ctFunc(funcName, varargin)
+function output = call(funcName, varargin)
```

Comment:
I really like what the namespaces accomplish here. However, I'd suggest to rename so we get the very intuitive `ct.clib.call` rather than `ct.impl.call`. I am aware that this is slightly inexact as it ignores the MATLAB wrapper, but we already added the `m` prefix to differentiate, and improve the docstring below.